### ISSUE TITLE
input = torch.cat(input, 1) in PoseExtNet.py

### ISSUE THREAD

**Issue by TimingSpace** [ClementPinard/SfmLearner-Pytorch#16]
Why concatenate the input in dimension 1？ If we concatenate in channel dimension, the dim value should be 1.
